models/chainlink/bnb/chainlink_bnb_price_feeds_hourly.sql

```diff
@@ -0,0 +1,124 @@
+{{ config(
+    alias = 'price_feeds_hourly',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['blockchain', 'hour', 'proxy_address', 'aggregator_address', 'underlying_token_address'],
```

Comment:
Questions:
1. `partition_by = ['block_date']` or `partition_by = ['hour']`
2. if `partition_by = ['block_date']`, should `'block_date'` go into unique_key?

Reply:
`block_date` is good for partition -- it does not need to be in unique keys just for the partitioning.